python/tests/phonenumberutiltest.py

```diff
@@ -2978,19 +2978,19 @@ def testIsMobileNumberPortableRegion(self):
         self.assertFalse(phonenumbers.is_mobile_number_portable_region("XY"))
 
     def testGetMetadataForRegionForNonGeoEntity_shouldBeNull(self):
-        self.assertTrue(PhoneMetadata.metadata_for_region("001") is None)
+        self.assertIs(PhoneMetadata.metadata_for_region("001"), None)
```

Comment:
(Note to self: this [requires Python 2.7](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIs), which is probably OK at this point &hellip;)

LGTM, just a couple of things:
- Could you apply the same change to all of the `self.assertTrue(... is None)` instances?  I think there's another dozen or so.
- Maybe shift to `self.assertIsNone` for better clarity?